### ISSUE TITLE
[Workaround] Simulating TB blur effect in Win11 22H2

### ISSUE THREAD

**Issue by e-t-l** [TranslucentTB/TranslucentTB#922]
Per #905, Windows version update 22H2 has removed the native "blur" translucency effect, and there is no way to reenable or reinstall it. This is a workaround.

1. Locate the image file location for your desktop background (or folder location, if you use Slideshow). You can find this by going to `Windows Settings > Personalization > Background`. If you use Spotlight as your background, follow [these instructions](https://www.lifewire.com/how-to-find-windows-spotlight-images-4776282) to find the location.
2. Download this Photoshop Action file: [BlurTB.atn.webm](https://user-images.githubusercontent.com/40775958/265144445-53a267b7-264e-43d2-9919-4f94cb26b67e.webm), and rename it to remove the ".webm" extension. This is not actually a WEBM file; Github just doesn't allow uploading .atn filetypes.
3. Import the Action file into Adobe Photoshop. (If you do not have Photoshop, go to [Photopea.com](https://photopea.com) and import it via `File > Open`
5. Open your background image in Photoshop/Photopea and run either action. If you use Slideshow or Spotlight, you will want to run this as a batch on the parent folder via `File > Automate > Batch...`
6. You will probably need to reconfigure some commands in the action for your own device configuration. For example, my native resolution is 1366x768, my taskbar height is 48px, and my taskbar location is Bottom, so those are the values I used. You can also just run the action, then resize/move the layer containing the clipping mask until it's in the right place.
7. You can also manually change the Gaussian blur radius value to increase or decrease the amount of blurriness.
8. Save the output file, and set your Windows desktop background to this new file.
9. Set TranslucentTB to Clear.

Basically, this process creates a copy of your current background image(s) and embeds a blurred overlay in the part of the image that your Windows Taskbar normally covers. When you set TranslucentTB to Clear, it will _appear_ as if the Taskbar is creating a translucent blur over the desktop background. In reality, this is just the blurred portion of the desktop background image that is fully visible. (Note: this means that hiding the taskbar will _not_ hide the blur effect!)

**Comment by EnderPlays25** [comment]
Can't download the webm file, it opens a broken video but no way to download it
